srcpkgs/zenity/template

```diff
@@ -13,7 +13,7 @@ license="LGPL-2.0-or-later"
 homepage="https://help.gnome.org/users/zenity/"
 changelog="https://gitlab.gnome.org/GNOME/zenity/-/raw/master/NEWS"
```

Comment:
Comment this, and add `changelog="https://gitlab.gnome.org/GNOME/zenity/-/raw/zenity-4-0/NEWS"` below.